app/routes/wow.shopping-list.tsx

```
@@ -63,44 +77,114 @@
+}
+
+type LoaderResponseType =
+  | {}
+  | { exception: string }
+  | (WoWListResponse & {
+      sortby: string
+      formValues: { itemID: number; maxPurchasePrice: number }
+    })
+
 type ActionResponseType =
   | {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)